frontend/javascripts/oxalis/model/sagas/proofread_saga.ts

```diff
@@ -121,6 +129,26 @@ function* loadCoarseMesh(layerName: string, segmentId: number, position: Vector3
   coarselyLoadedSegmentIds.push(segmentId);
 }
 
+function* checkForAgglomerateSkeletonModification(
+  action: CreateNodeAction | DeleteNodeAction | SetNodePositionAction,
+): Saga<void> {
+  let nodeId, treeId;
+
+  if (action.type === "CREATE_NODE") {
+    ({ treeId } = action);
+  } else {
+    ({ nodeId, treeId } = action);
+  }
+
+  const skeletonTracing = yield* select((state) => enforceSkeletonTracing(state.tracing));
+
+  getNodeAndTree(skeletonTracing, nodeId, treeId, TreeTypeEnum.AGGLOMERATE).map((_) => {
+    Toast.error(
+      "Agglomerate skeletons cannot be modified, except using the proofreading tool to add or delete edges.",
```

Comment:
I'd rephrase it to be "positive":

```suggestion
      "Agglomerate skeletons can only be modified when using the proofreading tool to add or delete edges.",
```